scripts/build-container.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+podman build --tag ibis:git ..
```

Comment:
As this is only a single command I would rather put it in the readme. Also Im not familiar with podman, I suppose the Containerfile also works with Docker? If so would be worth including the docker command as well.